tests/functional/registry/test_deployment.py

```diff
@@ -88,3 +95,14 @@ def test_experimental_deployments(
     )
     registry.endorseVault(experimental_vault, {"from": gov})
     assert registry.latestVault(token) == experimental_vault
+
+
+def test_governance(guardian, rewards, registry, create_token, create_vault, rando):
+    # Not just anyone can create a new endorsed Vault, only governance can!
+    with brownie.reverts():
+        registry.newVault(create_token(), guardian, rewards, "", "", {"from": rando})
```

Comment:
This seems better under `test_deployment_management` because only governance can create a new proxy deployment using `newVault`

Reply:
Makes sense. Moved.